clients/python-wrapper/tests/utests/common.py

```diff
@@ -0,0 +1,77 @@
+import importlib
+import os
+from contextlib import contextmanager
+from copy import deepcopy
+
+import lakefs_sdk
+import lakefs.repository
+from lakefs import config as client_config
+
+TEST_SERVER = "https://test_server"
+TEST_ACCESS_KEY_ID = "test_access_key_id"
+TEST_SECRET_ACCESS_KEY = "test_secret_access_key"
+TEST_CONFIG = f'''
+server:
+  endpoint_url: {TEST_SERVER}
+
+credentials:
+    access_key_id: {TEST_ACCESS_KEY_ID}
+    secret_access_key: {TEST_SECRET_ACCESS_KEY}
+'''
+
+TEST_CONFIG_KWARGS: dict[str, str] = {
+    "username": "my_username",
+    "password": "my_password",
+    "host": "http://my_host",
+    "access_token": "my_jwt_token"
+}
+
+TEST_ENDPOINT_PATH = "/api/v1"
```

Comment:
I don't think this should be part of the wrapper package

Reply:
It's not part of the wrapper package - it's part of the test package